packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3315,6 +3341,7 @@ export default class Meeting extends StatelessWebexPlugin {
             });
           }
         })
+        .then(this.handleMediaLogging)
```

Comment:
We still need to add the devicechange listner for the devices

Reply:
Where should this be placed?